app/src/main/java/org/simple/clinic/summary/teleconsultation/messagebuilder/TeleconsultationSmsMessageBuilder.kt

```diff
@@ -0,0 +1,19 @@
+package org.simple.clinic.summary.teleconsultation.messagebuilder
+
+import org.simple.clinic.summary.PatientTeleconsultationInfo
+import javax.inject.Inject
+
+class TeleconsultationSmsMessageBuilder @Inject constructor() : TeleconsultationMessageBuilder {
```

Comment:
We can still send the long version of the message over SMS; the only reason we are doing it in this way is because we want to fit into the character limit for a single SMS.

What do you think of naming these message builders, `LongMessageBuilder` and `ShortMessageBuilder` instead? We can separate the length of the message from the medium used as the communication channel.

Reply:
Yes, that makes sense but the reason I went with this approach is, it seems more common to have different messages depending on the client. For example, we might have a completely different long message if it is an email. WDYT? or should I go ahead and change it to `LongMessageBuilder` & `ShortMessageBuilder`?

Reply:
What I mean is that a class should not have any idea about how it is going to be used. For the message builder itself, whether the message that it constructs is sent via Email, SMS, Whatsapp, or anything else is irrelevant since that decision is taken elsewhere.

So adding the communication channel to the class name adds this knowledge to the class while the class itself doesn't do anything with that knowledge internally.

Reply:
Okay 👍🏾